Toolset/palettes/script editor/behaviors/revseeditorbehavior.livecodescript

```diff
@@ -4382,11 +4386,13 @@ private function findListObjectsOnAllAvailableStacks pCallback, pCallbackTarget,
    
```

Comment:
You could probably make this slightly more efficient by populating the `tMainstacks` variable with either `the mainstacks` or  `ideUserMainStacks()` depending on `gRevDevelopment`, then both the conditions in the repeat loop could go.

I wonder if it should be using `ideMainStacks()` instead anyway, which does the above but based on the `cShowRevolutionStacks` preference instead of `gRevDevelopment`. What do you think?